cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/views/tool/index.js

```diff
@@ -0,0 +1,132 @@
+      </div>
+      <div className="o-tool-card-group">
+        <div className="m-tool-card">
+          {" "}
+          <TotalPurchaseAmount />
+        </div>
+        <div className="m-tool-card">
+          {" "}
+          <DownPayment />
+        </div>
+        <div className="m-tool-card">
+          {" "}
+          <CreditApr />
+        </div>
+        <div className="m-tool-card">
+          {" "}
+          <MonthlyPayment />
+        </div>
+
+        <div className="o-tool-card-group__gray">
+          <h3>Deals and promotions</h3>
+          <DealOrPromo
+            num="1"
+            title="Rebate or coupon"
+            desc="Is the store offering an amount of money off the purchase price?"
+          />
+          <DealOrPromo
+            num="2"
+            title="Zero or reduced interest promotion"
+            desc="Is there an interest rate promotional period?"
+          />
+          <DealOrPromo
+            num="3"
+            title="Deferred interest promotion"
+            desc="IAre you being offered 0% interest if paid in full within a certain
+        number of months?"
+          />
+        </div>
+        <div className="m-tool-card">
+          <TrueCostBreakdown />
+        </div>
+        <div className="m-tool-card">
+          <AlternatePayoffScenarios />
+        </div>
+        <div className="m-tool-card">
+          <TipsAndResources />
+        </div>
+      </div>
+      {/* <DollarSliderInput
+        title="Total purchase amount"
+        description="This is..."
+      />
+      <PercentSliderInput title="Credit APR" description="This is.." />
+      <DollarSliderInput title="Credit APR" description="This is..." />
+      <MonthSliderInput
+        title="Total purchase amount"
+        description="This is..."
+      />
+      <h2>Special financing</h2>
+      <DollarInput title="Down payment" description="What amount..." />
+      <PercentForMonthsInput
+        title="Introductory.."
+        description="If there's a special..."
+      />
+      
+      <h2>Alternate payoff scenarios</h2>
+      <p className="description">Here are some other...</p>
+      <ScenarioTable
+        totalPurchaseAmount="$1000"
+        creditAprPct="6%"
+        monthlyPayment="$42.00"
+        monthsToPayOff="12"
+        stepMonths={2}
+      />
+      <h2>Now that you've found...</h2>
+      <RadioButtonQuestion
+        title="Can you comfortably..."
+        options={[
+          { value: "yes", label: "Yes" },
+          { value: "no", label: "No" },
+          { value: "maybe", label: "I'm not sure" }
+        ]}
+      />
+      <NonPaymentConsequences />
+      <Habits />
+      <Decision />
+      <h2>How is this calculated?</h2>
+      <p className="description">You are charged interest...</p>
+      <SectionTip text="Your are charged interest..." />
+      <SectionTip text="Your are charged interest..." />
+      <SectionTip
+        label="Resource"
+        text={
+          <>
```

Comment:
This is not idiomatic React. If you need to render a fragment, you should just be using `props.children` and putting stuff inside of the component rather than feeding it in as a prop.

e.g.

```jsx
<SectionTip label="Resource">
  You are charged interest...
  <Link to="/resource_url">Here's a link to the resource</Link>
</SectionTip>
```

You'll need to edit the `SectionTip` component to make this work, replacing `text` or `props.text` with `children` or `props.children` depending on how you're referring to the component's props.

Reply:
I took this out.